src/main/java/com/nus/iss/tasktracker/config/WebMvcConfig.java

```diff
@@ -32,7 +32,7 @@ public void addInterceptors(InterceptorRegistry registry) {
     public void addCorsMappings(CorsRegistry registry) {
         System.out.println("addCorsMappings called");
         registry.addMapping("/**")  // Allows CORS requests to all endpoints
-                .allowedOrigins("http://165.22.100.234:3000")  // Replace with the IP of your React app
+                .allowedOrigins("http://localhost:3000")  // Replace with the IP of your React app
```

Comment:
just make sure to adjust it for production or, ideally it should be configurable.